### ISSUE TITLE
Add option to reset analytics tracking id to Settings panel

### ISSUE THREAD

**Issue by nh758** [sillsdev/bible-karaoke#83]
To comply with google analytics policy.

**Comment by grahammcculloch** [comment]
@nh758 can you elaborate on what it means to 'reset analytics tracking id'? What action should take place in the code when the user clicks on said reset button in the Settings panel?

**Comment by nh758** [comment]
The tracking id is a locally stored identifier that is sent with the analytics hits to identify the same install over multiple sessions. GA policy requires this to be user-reset-able. The library we're using didn't handle the identifier in line with this, so I forked and modified it. I've made a pull request over a month ago, but doubt it will be accepted. I'm planning to publish it as a separate npm package. The reset button in the settings panel will make a call to the modified library that generates and stores the new tracking id. I can handle this on Friday.
